core/src/main/scala/com/salesforce/op/filters/Summary.scala

```diff
@@ -35,18 +35,36 @@ import com.twitter.algebird.Monoid
 /**
  * Class used to get summaries of prepared features to determine distribution binning strategy
  *
- * @param min minimum value seen
- * @param max maximum value seen
+ * @param min minimum value seen for double, minimum number of tokens in one text for text
+ * @param max maximum value seen for double, maximum number of tokens in one text for text
+ * @param sum sum of values for double, total number of tokens for text
+ * @param count number of doubles for double, number of texts for text
  */
-private[op] case class Summary(min: Double, max: Double)
+private[op] case class Summary(min: Double, max: Double, sum: Double, count: Double) {
+  /**
+   * to calculate the hashing size for RFF (compare js distance) for Text
+   * @param bins default bins of RFF
+   * @return
+   */
+  def getBinsText(bins: Int): Int = bins
```

Comment:
The user should be able to pass the formula for how to compute the text feature bin size into RawFeatureFilter as a parameter. We should provide a default which for now can be no formula but later can be updated

Reply:
Done.